recipe/meta.yaml

```diff
@@ -30,11 +26,11 @@ requirements:
 
 test:
   commands:
-    - test -f "${PREFIX}/lib/libunwind.so"
-    - test ! -f "${PREFIX}/lib/libunwind.a"
+    - test -f "${PREFIX}/lib/libunwind.so"  # [x86_64] 
+    - test -f "${PREFIX}/lib64/libunwind.so"  # [ppc64le] 
```

Comment:
This is wrong, shared libraries should always go into `lib` on conda-forge.

Reply:
Should be fixed now.